action-templates/actions/action-maven-release/action.yml

```diff
@@ -77,7 +77,7 @@ runs:
         git config --global user.email "github-actions@github.com"
         git config --global user.name "GitHub Actions"
         # get maven artifact id and provide as output
-        MVN_ARTIFACT_ID=$(mvn -f .${{inputs.app-path}}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)
+        MVN_ARTIFACT_ID=$(mvn -f ./${{inputs.app-path}}/pom.xml org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.artifactId -q -DforceStdout)
```

Comment:
https://github.com/it-at-m/refarch-templates/blob/main/.github/workflows/maven-release.yml#L34 also dann muss hier der slash weg. 

dann sind wir wieder bei dem hier https://github.com/it-at-m/lhm_actions/issues/42

Reply:
Genau, wobei das auch kein Problem ist, da das auch wieder nur zu `//` führt und damit nicht breaking ist.

Reply:
https://github.com/it-at-m/sps/actions/runs/14610509740/job/40987551008

du hast recht man kann tatsächlich // schreiben

Reply:
sorry falsch verstanden den mr. also das war die einzigste stelle die inkonsistent ist